src/actions/Intel/authActions.js

```diff
@@ -0,0 +1,65 @@
+//#region > Authentication
+/**
+ * Handle login
+ *
+ * @param user A user to login with
+ * @description Handles states for login
+ */
+const login = (user) => {
+  return async (intel) => {
+    try {
+      const session = intel.snekclient.session;
+
+      return await session.begin(user).then((whoami) => {
+        if (whoami?.username !== process.env.REACT_APP_ANONYMOUS_USER) {
+          return {
+            username: whoami.username,
+            avatarUrl:
+              "https://www.clipartmax.com/png/full/166-1669056_the-20-cooler-octocat-github-octocat.png",
```

Comment:
It is wise to use such a big picture?

32x32px should be enough.

Reply:
> It is wise to use such a big picture?
> 
> What about this picture? It gives me less anxiety than the other one.
> 
> <img alt="" height="200" src="https://camo.githubusercontent.com/7be1ef51e492043184abe241cdd8e6b2dc9b3af8/68747470733a2f2f656e637279707465642d74626e302e677374617469632e636f6d2f696d616765733f713d74626e253341414e643947635154394641586239454d65554c77777a544a37636837436b78347171367831574d49696c4f566a61417458534d4b476c533826757371703d434155">
> 
> 32x32px should be enough.

Looks nice xD. We should take this one.

Reply:
Change will be implement if you provide me a link that is shorter than:
"https://camo.githubusercontent.com/7be1ef51e492043184abe241cdd8e6b2dc9b3af8/68747470733a2f2f656e637279707465642d74626e302e677374617469632e636f6d2f696d616765733f713d74626e253341414e643947635154394641586239454d65554c77777a544a37636837436b78347171367831574d49696c4f566a61417458534d4b476c533826757371703d434155"

Reply:
32x32 is too small. Use 256x256 jpeg for now per favor. @Aichnerc's octocat is 225x225.